qiskit/pulse/timeslots.py

```diff
@@ -254,3 +267,9 @@ def __eq__(self, other) -> bool:
         if self.timeslots == other.timeslots:
             return True
         return False
+
+    def __repr__(self):
+        rep = dict()
+        for key, val in self._table.items():
```

Comment:
Is the purpose of this loop to eliminate the text `Interval` from the repr?

Reply:
Yes, the purpose was to get rid of the `Interval` text.

Reply:
Is it possible to use the string `replace` method for this, to make your intentions explicit? `str(self._table.items()).replace('Interval', '')`

Reply:
Using replace could be dangerous as it is introducing a dependency on the representation of the underlying items, it may be safer to duplicate the representation of the underlying items without Interval in this __repr__.

Reply:
`str(self._table.items()).replace('Interval', '')` prints the type of `self._table` as well, and it also prints the items in a list of tuples
```
TimeslotCollectiondict_items([(AcquireChannel(0), [(0, 10), (10, 20)]), 
                              (AcquireChannel(1), [(10, 20)])])
```

I tried changing `rep[key] = [(interval.begin, interval.end) for interval in val]` to `rep[key] = str(val).replace("Interval", '')`, which works except it prints the `Interval`s for each `Timeslot` as a string
```
TimeslotCollection{AcquireChannel(0): '[(0, 10), (10, 20)]', 
                   AcquireChannel(1): '[(10, 20)]'}
```
which still gets the representation across, but is not consistent with `Timeslot`'s representation

Reply:
Okay! I wasn't sure.